src/main/java/io/stargate/sgv2/jsonapi/service/shredding/collections/WritableShreddedDocument.java

```diff
@@ -302,20 +301,10 @@ public void shredVector(JsonPath path, String base64Vector) {
       // vector data is added only to queryVectorValues and exists keys index
       addKey(path);
 
-      byte[] binaryPayload;
       try {
-        binaryPayload = Base64Util.decodeFromMimeBase64(base64Vector);
-      } catch (IllegalArgumentException e) {
-        throw ErrorCodeV1.SHRED_BAD_BINARY_VECTOR_VALUE.toApiException(
-            "Invalid content in EJSON $binary wrapper: not valid Base64-encoded String, problem: %s",
-            e.getMessage());
-      }
-
-      try {
-        queryVectorValues = CqlVectorUtil.bytesToFloats(binaryPayload);
-      } catch (IllegalArgumentException e) {
-        throw ErrorCodeV1.SHRED_BAD_BINARY_VECTOR_VALUE.toApiException(
-            "Invalid content in EJSON $binary wrapper, problem: %s", e.getMessage());
+        queryVectorValues = EJSONWrapper.toFloatArray(base64Vector);
```

Comment:
is this change necessary ? It is essentially calling the same functions, and it is may not be  generating less useful errors ? 

Is this used when using binary vectors for sorts ?  I would prefer this change happened working with tatu if it is not needed